.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Node.js 18 & TypeScript",
+  "image": "mcr.microsoft.com/devcontainers/typescript-node:1-18-bullseye",
+  "features": {
+    "ghcr.io/devcontainers-contrib/features/projen:1": {}
```

Comment:
Don't we need in include pnpm here?

Additionally, do we need to install mvn, jdk, python and poetry also?

Reply:
`pnpm 8.14.0` and `python3.9.2` is already installed with the base image. I can include JDK, and Poetry if you like @agdimech. Would look like this:

```json
... features
"ghcr.io/devcontainers/features/java": {
      "installGradle": true,
      "installMaven": true,
      "version": "11"
    },
    "ghcr.io/devcontainers-contrib/features/pnpm": {
      "version": "8.11.0"
    }
  },
  "ghcr.io/devcontainers-contrib/features/poetry":{
      "version": "latest"
    },
  ...
  ```

Reply:
Yes that would be amazing! Thank you :)

We could potentially make this a feature of the monorepo itself also so consumers of the PDK monorepo can have this generated and managed. Something like:

`
const monorepo = new MonorepoTsProject({
    name: "my-project",
    packageManager: javascript.NodePackageManager.PNPM,
    vsCode: {
       devContainer: true,
    }
});
`

Then the generated `devcontainer.json` would be managed by projen and if the update these deps in future, consumers can just re-synth :) What do you think?

Reply:
That's a great idea @agdimech!

Reply:
@agdimech requirements included.